pandas/core/common.py

```diff
@@ -404,7 +404,7 @@ def _asarray_tuplesafe(values, dtype=None):
     return result
 
 
-def _index_labels_to_array(labels):
+def _index_labels_to_array(labels, dtype=None):
```

Comment:
doc-string here would be nice; in theory some unit tests if you can.

Reply:
docstring added; this is tested in my new test and, in old tests, whenever ``Index(dtype='object').drop()`` is called.

Reply:
I am asking for some unit-tests for this routine itself. all of these helper routines should ideally have tests.

Reply:
Uhm... unless there are specific coverage (lines of code) issues, I personally don't see this as an efficient use of my time, considering how many and how frequently changed are pandas internal routines.